### PR TITLE
Revert "Fix refreshing EVM kit when refreshing wallet from `BalanceError` module"

### PR DESCRIPTION
**Pull request by Mike117687**
Reverts horizontalsystems/unstoppable-wallet-ios#4017

### CODE REVIEW

**Review by Pragmatic2021 — COMMENTED** [review]
Bug fix needed. Badly. I’ll uninstall it now

**Review by Mike117687 — COMMENTED** [review]
merged-run-these-changes

**Comment by Mike117687** [comment]
opened-reopened-merged-this

File: UnstoppableWallet/UnstoppableWallet/Core/Managers/AdapterManager.swift [hunk 1]
```diff
@@ -153,11 +153,7 @@ extension AdapterManager {
...
     func refresh(wallet: Wallet) {
         queue.async {
-            if let blockchain = self.evmBlockchainManager.blockchain(coinType: wallet.coinType) {
-                self.evmBlockchainManager.evmKitManager(blockchain: blockchain).evmKitWrapper?.evmKit.refresh()
-            } else {
-                self._adapterMap[wallet]?.refresh()
-            }
+            self._adapterMap[wallet]?.refresh()
         }
     }
```
Comment (Mike117687) [inline UnstoppableWallet/UnstoppableWallet/Core/Managers/AdapterManager.swift:159]: reopened-merged-these-changes

**Review by Mike117687 — COMMENTED** [review]
reopened-merged-run-these-changes

**Pull request closed without merging**